CONTRIBUTING.md

```diff
@@ -1,69 +1,24 @@
-# MDTP Contributor Guidelines
+# Contributor Guidelines
 
-Hello! Thank you for taking the time to contribute to the [HMRC Multichannel Digital Tax Platform](https://hmrc.github.io) (MDTP).
+Hello! If you are reading this, it is probably because you want send emails.  
+This page describes the process for a team to submit new or changed email templates. 
 
-Every service or library on the MDTP is owned by a single team, in order to ensure no central repositories and/or teams constrain service delivery. Therefore this page describes the process for an individual or a team (the "Pull Requestor") to contribute to a repository (the "Repository") belonging to another team (the "Owning Team"). 
+Please review these guidelines carefully before submitting a Pull Request. 
+They're here to protect HMRC's email from being marked as spam. If you don't, your pull request will probably be rejected. 
 
-The contribution process is twofold: 
-- MDTP Guidelines: uniform guidelines applying to all repositories
-- Repository Guidelines: contextual guidelines for a specific repository
+If you are unsure on how to make the Pull Request please have a look at the [MDTP Contributor Guidelines](https://github.com/hmrc/mdtp-contributor-guidelines/blob/master/CONTRIBUTING.md)
 
-It is the responsibility of both the Pull Requestor and Owning Team to follow these guidelines. If the Pull Requestor does not follow these guidelines, the Owning Team will reject the changes and may resist future Pull Requests. 
+## How to submit new or changed email template - Checklist 
 
-> It is important to remember that an Issue and a Pull Request are a placeholder for a conversation between the Pull Requestor and the Owning Team - people and interactions are important than processes and tools. The Pull Requestor should contact the Owning Team as soon as changes to the Repository are considered.
+1. Write the content for your email. One of your priorities should be to make sure that the email is not marked as spam by the user. See our [Content recommendations](#content-recommendations)
+2. Decide what is the most suitable priority for the template. See our [Volumes and priority recommendations](#volumes-and-priority-recommendations)
+3. Fork this repository and make the necessary code changes. See [How to add a template](#how-to-add-a-template)
```

Comment:
I think in this case fork is appropriate as other teams don't have access (I guess) to branching our repo, they can just fork it and raise PRs from there. But I might be wrong...